test/Datadog.Trace.Tests/Logging/NLogLogProviderTests.cs

```diff
@@ -3,24 +3,33 @@
 using Datadog.Trace.Logging.LogProviders;
 using NLog;
 using NLog.Config;
+using NLog.Layouts;
 using NLog.Targets;
 using Xunit;
 
 namespace Datadog.Trace.Tests.Logging
 {
     [Collection(nameof(Datadog.Trace.Tests.Logging))]
+    [TestCaseOrderer("Datadog.Trace.TestHelpers.AlphabeticalOrderer", "Datadog.Trace.TestHelpers")]
```

Comment:
Not ideal, but I used a test case orderer so I could avoid a bug that occurs when running two tracer's sequentially with different DD_LOGS_INJECTION settings.